UpSkill_App/UpSkill.Infrastructure/Data/Models/Course.cs

```diff
@@ -1,24 +1,35 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace UpSkill.Infrastructure.Data.Models
+﻿namespace UpSkill.Infrastructure.Data.Models
 {
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+
     public class Course
     {
         [Key]
+        [Required]
         public string Id { get; init; } = Guid.NewGuid().ToString();
 
         [Required]
         public string Name { get; set; }
 
+        [Required]
+        public string Description { get; set; }
+
+        [Required]
+        public string Category { get; set; }
```

Comment:
I think by design we might have to introduce category model or at least enum.

Reply:
I agree although I am not so sure what the Category model should contain. I created a Category model which contains an ID and a name. If you have ideas about the contents of the model, please tell me.